app/forms/citizens/other_assets_form.rb

```diff
@@ -38,7 +38,7 @@ class OtherAssetsForm
 
     attr_accessor(*ALL_ATTRIBUTES)
     attr_accessor(*CHECK_BOXES_ATTRIBUTES)
-    attr_accessor :check_box_second_home, :check_box_none_selected, :mode
+    attr_accessor :mode
```

Comment:
What's `mode`. This looks suspiciously like `journey` in `SavingsAmountsForm`. ...... hang on ..... Yes, it should be `journey`. The inconsistency appears across a number of controllers and forms. Bum. I'll raise a tech debt ticket to fix that.

Reply:
https://dsdmoj.atlassian.net/browse/AP-917